Samples/AccountTransfer.NetCore/NuGet.Config

```diff
@@ -1,7 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <!-- <add key="Orleans-local" value="..\..\Artifacts\Debug\" /> -->
-    <add key="Orleans-myget" value="https://dotnet.myget.org/F/orleans-prerelease/api/v3/index.json" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
```

Comment:
should we remove this nugget.config file here too? since I think all samples shouldn't be using myget